firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -67,9 +67,8 @@ homing_positive_dir: true
 
 [tmc2209 stepper_x]
 uart_pin: PE6
-interpolate: True
+interpolate: False
```

Comment:
Apologies, one thing I forgot to mention is that if you turn interpolate off, you'll want to increase the microsteps to 32 or else the printer is going to be REALLY loud. It's up to you if you want to change them or keep it as-is, just let me know either way.

Reply:
I've left mine as 16 without interpolate and haven't had an issue (I run 4 on my Zs though and *that* is loud AF).  I'd rather leave it at 16 since some people use 0.9° steppers and 32 microsteps is getting silly.